gnosis/src/main/scala/edu/uci/ics/cloudberry/gnosis/IGeoIndex.scala

```diff
@@ -41,4 +41,32 @@ class USGeoJSONIndex() extends IGeoIndex {
   override def search(envelope: Envelope): Seq[IUSGeoJSONEntity] = {
     index.query(envelope).asScala.map(item => entities(item.asInstanceOf[Int]))
   }
+}
+
+class NYGeoJSONIndex() extends IGeoIndex {
```

Comment:
It can't be an instantiated object of USGeoJsonIndex, because they use different entities.